[2025-02-09T18:05:18.100+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-09T18:05:18.128+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_postgres_and_csv_data.run_load_data_script manual__2025-02-09T18:05:04.110541+00:00 [queued]>
[2025-02-09T18:05:18.140+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_postgres_and_csv_data.run_load_data_script manual__2025-02-09T18:05:04.110541+00:00 [queued]>
[2025-02-09T18:05:18.141+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-09T18:05:18.327+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): run_load_data_script> on 2025-02-09 18:05:04.110541+00:00
[2025-02-09T18:05:18.337+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=389) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-09T18:05:18.340+0000] {standard_task_runner.py:72} INFO - Started process 391 to run task
[2025-02-09T18:05:18.340+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_postgres_and_csv_data', 'run_load_data_script', 'manual__2025-02-09T18:05:04.110541+00:00', '--job-id', '232', '--raw', '--subdir', 'DAGS_FOLDER/extract_postgres_and_csv_data.py', '--cfg-path', '/tmp/tmp2nd2jfyl']
[2025-02-09T18:05:18.342+0000] {standard_task_runner.py:105} INFO - Job 232: Subtask run_load_data_script
[2025-02-09T18:05:18.403+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_postgres_and_csv_data.run_load_data_script manual__2025-02-09T18:05:04.110541+00:00 [running]> on host 58bc83bc12fd
[2025-02-09T18:05:18.511+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_postgres_and_csv_data' AIRFLOW_CTX_TASK_ID='run_load_data_script' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T18:05:04.110541+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-09T18:05:04.110541+00:00'
[2025-02-09T18:05:18.513+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-09T18:05:18.514+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-09T18:05:18.514+0000] {logging_mixin.py:190} INFO - Current task name:run_load_data_script state:running start_date:2025-02-09 18:05:18.129304+00:00
[2025-02-09T18:05:18.515+0000] {logging_mixin.py:190} INFO - Dag name:extract_postgres_and_csv_data and current dag run status:running
[2025-02-09T18:05:18.516+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-09T18:05:18.517+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-09T18:05:18.519+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/meltano/meltano_project/load/load_data_csv_for_db.py']
[2025-02-09T18:05:18.534+0000] {subprocess.py:99} INFO - Output:
[2025-02-09T18:05:21.156+0000] {subprocess.py:106} INFO - Conexão bem-sucedida ao banco de dados!
[2025-02-09T18:05:21.157+0000] {subprocess.py:106} INFO - Tabela 'order_details' criada com sucesso!
[2025-02-09T18:05:21.158+0000] {subprocess.py:106} INFO - Carregando dados do arquivo: /opt/***/meltano/data/csv/2025-02-09/order_details.csv
[2025-02-09T18:05:21.159+0000] {subprocess.py:106} INFO - Conexão bem-sucedida ao banco de dados!
[2025-02-09T18:05:21.160+0000] {subprocess.py:106} INFO - Dados inseridos com sucesso!
[2025-02-09T18:05:21.162+0000] {subprocess.py:106} INFO - Conexão bem-sucedida ao banco de dados!
[2025-02-09T18:05:21.163+0000] {subprocess.py:106} INFO - Erro ao buscar dados: No module named 'openpyxl'
[2025-02-09T18:05:21.327+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-09T18:05:21.368+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-09T18:05:21.370+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_postgres_and_csv_data, task_id=run_load_data_script, run_id=manual__2025-02-09T18:05:04.110541+00:00, execution_date=20250209T180504, start_date=20250209T180518, end_date=20250209T180521
[2025-02-09T18:05:21.403+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-09T18:05:21.404+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-09T18:05:21.405+0000] {logging_mixin.py:190} INFO - Dag name:extract_postgres_and_csv_data queued_at:2025-02-09 18:05:04.126434+00:00
[2025-02-09T18:05:21.406+0000] {logging_mixin.py:190} INFO - Task hostname:58bc83bc12fd operator:BashOperator
[2025-02-09T18:05:21.452+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-09T18:05:21.487+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-09T18:05:21.491+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
